src/libecalc/core/graph_result.py

```diff
@@ -631,6 +649,24 @@ def get_asset_result(self) -> libecalc.dto.result.EcalcModelResult:
                                     rate_type=RateType.STREAM_DAY,
                                     regularity=regularity.for_timesteps(model.timesteps).values,
                                 ),
+                                standard_rate_sm3_per_day=TimeSeriesRate(
+                                    timesteps=model.timesteps,
+                                    values=stage_result.inlet_stream_condition.standard_rate_sm3_per_day
+                                    if stage_result.inlet_stream_condition.standard_rate_sm3_per_day is not None
+                                    else [math.nan] * len(model.timesteps),
+                                    unit=Unit.STANDARD_CUBIC_METER_PER_DAY,
+                                    rate_type=RateType.STREAM_DAY,
+                                    regularity=regularity.for_timesteps(model.timesteps).values,
+                                ),
+                                standard_rate_before_asv_sm3_per_day=TimeSeriesRate(
+                                    timesteps=model.timesteps,
+                                    values=stage_result.inlet_stream_condition.standard_rate_before_asv_sm3_per_day
+                                    if stage_result.inlet_stream_condition.standard_rate_before_asv_sm3_per_day is not None
```

Comment:
```suggestion
                                    if stage_result.outlet_stream_condition.standard_rate_before_asv_sm3_per_day is not None
```

Reply:
It is standard rate, inlet and outlet should be same from material balance.

Reply:
Yes. I was thinking that this operation (inlet=outlet) should be solved in the CompressorTrainStage (stage.py), so that you always read inlet from inlet, and outlet from outlet in graph_result. Not sure, maybe it is ok:-) Thoughts @jsolaas?

Reply:
I agree, since we have both inlet and outlet we should map inlet -> inlet and outlet -> outlet. In this case I assume they are the same, but as Frode said, that operation is handled by CompressorTrainStage already

Reply:
@jsolaas @frodehk, the suggested changes had been updated in the new commits. Could you please take a review if they are OK and can be approved?